vignettes/daisyChains.Rmd

```diff
@@ -127,6 +127,21 @@ Here's a minified responsive timeline taken from https://daisyui.com/components/
 ```{r setup}
 library(daisyuiwidget1)
 
+# Using data frame format (recommended)
```

Comment:
The example is good but let's remove the old one.  Does `stringsAsFactors=FALSE` make any difference here?